### ISSUE TITLE
Registration process should include a link to user activation KB article

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/userdetails#65]
Currently the [user registration page](https://auth.ala.org.au/userdetails/registration/createAccount) includes a help paragraph indicating that an activation email will be sent. 

The text about email activation needs to be made prominent so users know they need it to use the account (TBC).

A link to the KB -  [I created an account but did not receive the account activation email](https://support.ala.org.au/support/solutions/articles/6000195196-i-created-an-account-but-did-not-receive-the-account-activation-email) needs to be added for further details. 

The KB needs to be enhanced with links on how to check for spam emails (and how to add support@ala.org.au to the safe senders list) for popular services: gmail, yahoo, hotmail? outlook.   Probably worth analysing  what are the most common email addresses for our current userbase.

Ticket #66 has changes that will impact this ticket as currently specified.

**Comment by Rita-C** [comment]
Hi @javier-molina @nickdos @vswan , do you have any suggestions?
1. > The text about email activation needs to be made prominent so users know they need it to use the account (TBC).

Make the `An "account activation" link will be emailed to the address provided` paragraph bold? 

2. > A link to the KB - [I created an account but did not receive the account activation email](https://support.ala.org.au/support/solutions/articles/6000195196-i-created-an-account-but-did-not-receive-the-account-activation-email) needs to be added for further details.

Add the link as part of above paragraph? Need some linking words.

3. > The KB needs to be enhanced with links on how to check for spam emails (and how to add [support@ala.org.au](mailto:support@ala.org.au) to the safe senders list) for popular services: gmail, yahoo, hotmail? outlook. Probably worth analysing what are the most common email addresses for our current userbase.

Is this something Verity or Alex can help with?

**Comment by matthewandrews** [comment]
I'd suggest adding (in bold and maybe red colour) **You will not be able to log in until your email address is confirmed.**  at the start of that second para under Your Account.   Then the same again on the post-submit page!

Also there is a typo: "You need click this link" should be "You need to click this link,".

**Comment by nickdos** [comment]
Do we want to look into the idea of not preventing login without confirmation (do social logins require it??)? A lot of sites do this... we could nag users when they login to confirm... maybe needs a bit of BA investigation...

**Comment by matthewandrews** [comment]
Yes that's the other option of course.  Delegated-authentication logins (Google/Facebook/Twitter) don't require it because we assume their email address has already been confirmed by the third party.  The downside would be bot accounts: we would need to add a (re?)CaPTCHA or similar.  Without a confirmed email address, obviously a user's attempts to reset a password won't work, and (as a minor thing) they won't get our newsletters.

**Comment by Rita-C** [comment]
> Do we want to look into the idea of not preventing login without confirmation (do social logins require it??)? A lot of sites do this... we could nag users when they login to confirm... maybe needs a bit of BA investigation...

#66 addresses this?

**Comment by nickdos** [comment]
Yes, #66 is the same thing.

**Comment by matthewandrews** [comment]
Hmm addressing the same issue but a delayed enforcement is not the same as not requiring it.  Either would need a (re)CaPTCHA on signup, but the delayed enforcement solution would need more work to add the nagging (and the delayed enforcement).

**Comment by Rita-C** [comment]
Registration page has a reCaPTCHA now, I agree that the delayed enforcement needs more work, e.g. database change, and it could confuse the user, i.e. my account stopped working.

**Comment by javier-molina** [comment]
@Rita-C, all

How about the new Your account section below

----------------------------------------------------------------------

### Your account
Your email address will be your ALA account login. Your email address is need it for [resetting your password](https://support.ala.org.au/support/solutions/articles/6000194958-my-username-password-isn-t-working-or-i-can-t-remember-my-username-password) and for [receiving communications](https://support.ala.org.au/support/solutions/articles/6000195858-how-do-i-manage-email-alerts-from-the-ala-) from us. 

**You will not be able to log in until your email address is confirmed.**

An "account activation" link will be emailed to the address provided. You need click this link, in order to complete the registration process. Note, you may need to check you spam/junk mail folder, as activation emails sometimes get caught by mail filters. You can find more information and further instructions on this [help article](https://support.ala.org.au/support/solutions/articles/6000195196-i-created-an-account-but-did-not-receive-the-account-activation-email).

----------------------------------------------------------------------

I have incorporated Matt's input except the text colour as if might be confused with a link.

I'm not sure whether there is a decision about exploring #66 or even implementing it in which case probably makes sense to close this, I would keep the conversation and decision about allowing login without email confirmation in #66 though.

**Comment by matthewandrews** [comment]
That looks good @javier-molina - I think we should also display that message on the post-submit page.

**Comment by Rita-C** [comment]
Thanks everyone who contributed ideas!
If no objections I'll implement the wording changes on registration page and post-submit page, leave #66 on hold.

**Comment by sbearcsiro** [comment]
Reviewed
